### ISSUE TITLE
Model giving nan loss in supervised setting.

### ISSUE THREAD

**Issue by arc-arnob** [mims-harvard/UniTS#34]
When trying to train in Crypto Dataset the model does not train properly and loss reported during training is always nan. I ran the model with other datasets as well, mostly related worked well but not for Crypto currency.

https://www.kaggle.com/competitions/g-research-crypto-forecasting/data?select=example_sample_submission.csv

**Comment by gasvn** [comment]
You can try reducing the learning rate and adjusting the batch size. Additionally, some data might be out of distribution, so you can identify such instances by analyzing the training loss.
